drivers/SmartThings/zwave-sensor/src/test/test_aeotec_multisensor_6.lua

```diff
@@ -219,96 +219,190 @@ test.register_coroutine_test(
           mock_sensor,
           SensorMultilevel:Get({sensor_type = SensorMultilevel.sensor_type.ULTRAVIOLET})
       ))
-      
+
       test.socket.zwave:__expect_send(zw_test_utils.zwave_test_build_send_command(
-          mock_sensor,                                                           
-          Configuration:Get({parameter_number = 9})                             
+          mock_sensor,
+          Configuration:Get({parameter_number = 9})
       ))
-      
 
     end
 )
 
 test.register_message_test(
   "SensorMultilevel report ultraviolet type should be handled as ultravioletIndex",
+  {
+    {
+      channel = "zwave",
+      direction = "receive",
+      message = { mock_sensor.id, zw_test_utils.zwave_test_build_receive_command(SensorMultilevel:Report({
+        sensor_type = SensorMultilevel.sensor_type.ULTRAVIOLET,
+        sensor_value = 10
+      })) }
+    },
     {
```

Comment:
Dang, the diff went a little crazy from this point on...

Reply:
yeah I'd turn off "display whitespace changes"